3drepo/3drepo.io
# Issue 1917 - Store Issues screenshots as external refs

This fixes #1917 

#### Description
Store screenshots externally via file storage the server is configured to connect to (fs, s3, gridfs etc?)

#### Test cases
- When creating an issue the viewpoints screenshots should be saved in a external ref
- When creating a risk the viewpoints screenshots should be saved in a external ref
- When creating a comment the viewpoint should work the same as when creating the issue/risk
- When showing the screenshot it should access the external ref in the new ones and the old ones should work as they do now
- When deleting a screenshot it should delete the new ones using the external ref and the old ones should work as they do now
- Check what happens on update
- Check that everything still works with BCF
